Resources/Prototypes/Entities/Structures/Doors/Firelocks/firelock.yml

```diff
@@ -37,7 +37,6 @@
         - state: closed_unlit
           shader: unshaded
           map: ["enum.DoorVisualLayers.BaseUnlit"]
-          visible: false
```

Comment:
By default, this shouldn't be visible and you shouldn't have to remove this line.